lib/puppet/application/apply.rb

```diff
@@ -333,11 +333,11 @@ def setup
   private
 
   def read_catalog(text)
+    format = Puppet::Resource::Catalog.default_format
     begin
-      catalog = Puppet::Resource::Catalog.convert_from(Puppet::Resource::Catalog.default_format,text)
-      catalog = Puppet::Resource::Catalog.pson_create(catalog) unless catalog.is_a?(Puppet::Resource::Catalog)
```

Comment:
Out of curiosity, I tried to trace back why we were doing this, and it seem slike this line has been dead code since 0cb5e7d226934e74f8b53dc1f6425b47f536afab. Does that sound accurate?